inference/pytorch_inference.py

```diff
@@ -21,18 +21,19 @@
 IMG_SIZE = 256
 CENTER_CROP_SIZE = 224
 NESTING_LIST=[2**i for i in range(3, 12)]
+NESTING_LIST=[8+(204*(i-1)) for i in range(1, 12)]
+print(NESTING_LIST)
```

Comment:
Add a print comment that makes it clear.

Reply:
This was leftover debugging code, removed in next patch